fast64_internal/sm64/sm64_constants.py

```diff
@@ -1768,3 +1768,216 @@ def __init__(self, geoAddr, level, switchDict):
     ("macro_yellow_coin", "Yellow Coin", "Yellow Coin"),
     ("macro_yellow_coin_2", "Yellow Coin 2", "Yellow Coin 2"),
 ]
+
+marioAnimations = [
+#   ( "animation name",                                             "adress" ),
+    ( "0 - Climb up ledge (right leg going up)",                    5162640  ),
```

Comment:
this file is formatted, so to keep things simple we should just keep it that way
I understand formatting breaks the column layout you have going on, so what about you swap the columns:
```py
marioAnimations = [
    # (address, "animation name"),
    (5162640, "0 - Climb up ledge (right leg going up)"),
    (5165520, "1 - Thrown from painting"),
    (5165544, "2 - Blown by wind"),
```
(this is after formatting)

Reply:
Also could you indicate which rom are these addresses for? unless all roms of sm64 are alike
(I'm a OoT enjoyer and there are many ROM versions from where I come 8) )